vars/pushDockerImage.groovy

```diff
@@ -23,16 +23,13 @@ def call(body) {
 
         container(name: 'tools') {
             withCurrentRepo { def repoUrl, def repoName, def repoOwner, def repoBranch ->
-                def dockerImage = "${dockerRegistryURL}/${repoOwner.toLowerCase()}/${repoName.toLowerCase()}"
-                def dockerImageVersion = stakaterCommands.getBranchedVersion("${versionPrefix}.${env.BUILD_NUMBER}")
+                def imageName = repoName.split("dockerfile-").last().toLowerCase()    
```

Comment:
Does it work if there's no "dockerfile-" prefix? Because we're using this in some repos that don't have this as a prefix

Reply:
I am going to test it